applications/DEMApplication/custom_constitutive/DEM_D_JKR_cohesive_law.cpp

```diff
@@ -38,13 +38,13 @@ namespace Kratos {
         const double radius_sum_inv = 1.0 / radius_sum;
         const double equiv_radius   = my_radius * other_radius * radius_sum_inv;
         const double contact_radius = sqrt(equiv_radius * indentation);
-        const double cohesive_force = equiv_young * sqrt(8.0 * equiv_cohesion * Globals::Pi * contact_radius * contact_radius * contact_radius / equiv_young);
+        const double cohesive_force = sqrt(8.0 * equiv_cohesion * Globals::Pi * contact_radius * contact_radius * contact_radius * equiv_young);
```

Comment:
Wow! This is a big change/bugfix!!

Reply:
so the first review on  const double cohesive_force ~  2*Pi*r
in the second review  const double cohesive_force ~  8*Pi*r^3
correct?